gatsby/content/projects/clients/syphon.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: projectimage
+title: Syphon
+categories:
+ - client
+description: a not for profit, open source matrix client with a focus on privacy and ease of use
```

Comment:
Upper case the A in "A not for profit..." and we should be good!

Reply:
Done. Marking as ready :D